bundles/multitenant-managed-api-service/1.21.0/manifests/multitenant-managed-api-service.clusterserviceversion.yaml

```diff
@@ -0,0 +1,747 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RHMI",
+          "metadata": {
+            "name": "INSTALLATION_NAME"
+          },
+          "spec": {
+            "deadMansSnitchSecret": "INSTALLATION_PREFIX-deadmanssnitch",
+            "namespacePrefix": "INSTALLATION_PREFIX-",
+            "operatorsInProductNamespace": "OPERATORS_IN_PRODUCT_NAMESPACE",
+            "pagerDutySecret": "INSTALLATION_PREFIX-pagerduty",
+            "priorityClassName": "INSTALLATION_SHORTHAND-pod-priority",
+            "selfSignedCerts": "SELF_SIGNED_CERTS",
+            "smtpSecret": "INSTALLATION_PREFIX-smtp",
+            "type": "INSTALLATION_TYPE",
+            "useClusterStorage": "USE_CLUSTER_STORAGE"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RHMI",
+          "metadata": {
+            "name": "example-rhmi"
+          },
+          "spec": {
+            "deadMansSnitchSecret": "redhat-rhmi-deadmanssnitch",
+            "namespacePrefix": "redhat-rhmi-",
+            "pagerDutySecret": "redhat-rhmi-pagerduty",
+            "priorityClassName": "rhoam-pod-priority",
+            "selfSignedCerts": true,
+            "smtpSecret": "redhat-rhmi-smtp",
+            "type": "managed",
+            "useClusterStorage": "true"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RHMIConfig",
+          "metadata": {
+            "name": "example-rhmiconfig"
+          },
+          "spec": {}
+        }
+      ]
+    capabilities: Basic Install
+    categories: Integration & Delivery
+    certified: "false"
+    containerImage: quay.io/integreatly/multitenant-managed-api-service:v1.21.0
+    olm.properties: '[{"type": "olm.maxOpenShiftVersion", "value": "4.10"}]'
```

Comment:
@carlkyrillos Can we change this to 4.10.15 and also look at how we might keep this up to date. It should be the same as what we've tested on nightly pipelines. Maybe it's part of the release process and manual.

Reply:
Yes will do :+1:

Reply:
@carlkyrillos 4.10.15 actually won't work. OLM will complain. So, 4.10 is fine .